### PR TITLE
add project and skill models

### PR DESCRIPTION
**Pull request by Bizzaro**

### CODE REVIEW

File: backend/app/models.py [hunk 1]
```diff
@@ -38,3 +35,17 @@ def to_dict(self) -> dict:
...
         }
+
+class Project(Base):
+    __tablename__ = "project"
+    project_id = Column(Integer, primary_key=True, index=True)
+    user_id = Column(Integer, ForeignKey("users.user_id"), nullable=False)
+    project_title = Column(String(100), nullable=False)
+    project_description = Column(String, nullable=False)
+    project_location = Column(String(50), nullable=False)
+    project_skillset = Column(ARRAY(Integer), ForeignKey("skill.skill_id"))
```
Comment (Celeo) [inline backend/app/models.py]: This functions as a many-to-many?

**Comment by Celeo** [comment]
image

File: backend/app/models.py [hunk 2]
```diff
@@ -18,3 +18,12 @@ class User(Base):
...
     user_last_login = Column(DateTime)
+
+class Project(Base):
+    __tablename__ = "project"
+    project_id = Column(Integer, primary_key=True, index=True)
+    user_id = Column(Integer, ForeignKey("users.user_id"))
+    project_title = Column(String(100))
+    project_description = Column(String)
+    project_location = Column(String(50))
+    project_skillset = Column(HSTORE)
```
Comment (petersteele111) [inline backend/app/models.py]: PEP8, please PEP8 :(

Comment (Bizzaro) [reply backend/app/models.py]: got a linter installed :)

**Comment by Bizzaro** [comment]
> SQLAlchemy isn't able to create the tables on my VM. Does it get up and running without issue on yours?

hmm let me nuke my db and try

**Pull request merged**